### DISCUSSION TITLE
Accessing Mouse MSigDB Collections

### DISCUSSION THREAD

**Discussion by agdenadel** [igordot/msigdbr#46] [Q&A] [answered]
Is there any possibility for the package to support collections that don't correspond to the human collections H, C1, ..., C8? For example accessing MH, M1, ..., M8 listed at the link below?

https://www.gsea-msigdb.org/gsea/msigdb/mouse/collections.jsp

**Comment by igordot** [comment]
Thank you for the suggestion. This is definitely a possibility.

See also: https://github.com/igordot/msigdbr/issues/25

**Comment by igordot** [comment, marked as answer]
The latest version of the msigdbr package is now on CRAN. It includes the 2024.1 version of human and mouse databases.
